saturnbase/Dockerfile.miniconda3-tips

```diff
@@ -0,0 +1,59 @@
+# https://jcrist.github.io/conda-docker-tips.html
+
+FROM continuumio/miniconda3:4.7.12 as builder
+
+RUN conda config --system --add channels conda-forge && \
+    conda install --yes --freeze-installed \
+    nomkl \
+    dask==2.10.1 \
+    numpy==1.18.1 \
+    pandas==1.0.1 \
+    tini==0.18.0 \
+    && conda clean -afy \
+    && find /opt/conda/ -follow -type f -name '*.a' -delete \
+    && find /opt/conda/ -follow -type f -name '*.pyc' -delete \
+    && find /opt/conda/ -follow -type f -name '*.js.map' -delete \
+    && find /opt/conda/lib/python*/site-packages/bokeh/server/static -follow -type f -name '*.js'
+
+COPY jupyter.yml /tmp
+
+# TODO: root?
+RUN conda env update -n root -f /tmp/jupyter.yml
```

Comment:
I think the issue is that you need to push all of jcrist optimizations into the conda env update step.

Reply:
I'm very wary of the multistage build because we're dealing with things that for example, link against a large variety of C++ libraries, and in the GPU case we link against CUDA, so I'm really not sure that switching to slim or any other base image  is a robust strategy